UnitTests/BTPaymentMethodNonce+DropInTests.swift

```diff
@@ -3,61 +3,40 @@ import BraintreeDropIn
 
 class BTPaymentMethodNonceDropInTests: XCTestCase {
 
-    let sharedParser = BTPaymentMethodNonceParser.shared()
-
     func testPaymentDescription_whenCardNonceWithLastTwo() {
-        let cardJSON = BTJSON(value: [
-            "nonce": "fake-nonce",
-            "description": "Visa ending in 11",
-            "details": [
-                "lastTwo" : "11",
-                "cardType": "visa"
-            ]
-        ])
-        let cardNonce = sharedParser.parseJSON(cardJSON, withParsingBlockForType: "CreditCard") as? BTCardNonce
+        class MockCardNonce: BTCardNonce {
```

Comment:
I created mock classes to work around the fact that the tests need to set properties that are read-only. It's not ideal, but I couldn't come up with a better option.